packages/apps-config/src/settings/endpoints.ts

```diff
@@ -147,7 +152,7 @@ function createTest (t: TFunction): LinkOption[] {
     },
     {
       info: 'nodle',
-      text: t<string>('rpc.arcadia', 'Arcadia (Nodle Testnet, hosted by Nodle)', { ns: 'apps-config' }),
+      text: t<string>('rpc.nodle-arcadia', 'Arcadia (Nodle Testnet, hosted by Nodle)', { ns: 'apps-config' }),
```

Comment:
Here you renamed the key, there has been translations done, so would just search for `rpc.arcadia` in packages/apps/public and just replace with the new.

Reply:
Got it!